en/6/00-overview.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: App Front-ends & Web3.js
-header: "Lesson 6: App Front-ends & Web3.js"
-roadmap: roadmap6.png
+title: Chainlink, Oracles, External Computation & Data
+header: "Lesson 6: Chainlink: The Decentralized Blockchain Oracle"
+roadmap: xxx
 path: solidity
 publishedOn: Cryptozombies
 ---
 
-Huh, you've made it this far?!
+You just keep getting more impressive by the second, but how will you fair when we look to combine the **real world** and all of it's changes and information with our beloved zombies?
 
-You're no ordinary CryptoZombie...
+In this lesson, we are going to go over **Chainlink** and **oracles**. 
 
-By completing Lesson 5, you've demonstrated that you have a pretty firm grasp of Solidity.
+[Oracles](https://betterprogramming.pub/what-is-a-blockchain-oracle-f5ccab8dbd72) or [Blockchain Oracles](https://betterprogramming.pub/what-is-a-blockchain-oracle-f5ccab8dbd72) are devices that connect our smart contracts and zombies with data and computation from the real world, such as [pricing data on currencies](https://data.chain.link/), [random number generators](https://docs.chain.link/docs/get-a-random-number/), and [any other data we can think of](https://docs.chain.link/docs/make-a-http-get-request/). Interacting outside of a blockchain isn't possible without an oracle, as blockchains themselves are intentionally isolated and deterministic by nature.  
```

Comment:
@PatrickAlphaC Just a note: markdown hyperlinks don't get rendered. Also, I edited this for clarity, as I think active voice sentences are easier to understand.